config.go

```diff
@@ -110,6 +130,25 @@ type config struct {
 	ProxyUser        string `long:"proxyuser" description:"Username for proxy server"`
 	ProxyPass        string `long:"proxypass" default-mask:"-" description:"Password for proxy server"`
 
+	// Automatic ticket buyer settings
```

Comment:
I think all of these settings can be removed, as long as we're ok with using a client to enable the ticket purchasing with the specific parameters.  For "easy" (parameterless) ticket buying these aren't needed.